### PR TITLE
[HER-76] Create detection parameter

### PR DESCRIPTION
**Pull request by ConstantinescuAndrei**
## Description
- it will leave inside Model Directory in Business
- it will have id,score,class,timestamp and PPagination as properties

## Related 
[HER-76](https://nerds-interns.atlassian.net/jira/software/projects/HER/boards/1?issueParent=10022&selectedIssue=HER-76)

### CODE REVIEW

File: src/Business/Detection/Common/Models/PDetection.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    {
+        public int Id { get; set; }
+
+        public int Score { get; set; }
+
+        public string Class { get; set; }
+
+        public DateTime Timestamp { get; set; }
+
+        public PPagination PPagination { get; set; }
```
Comment (danidoff) [inline src/Business/Detection/Common/Models/PDetection.cs:16]: Should we initialize it with `new PPagination`?

**Review by danidoff — CHANGES_REQUESTED** [review]

File: src/Business/Detection/Common/Models/PDetection.cs (hunk 1 above)
Comment (ConstantinescuAndrei) [reply src/Business/Detection/Common/Models/PDetection.cs]: done

**Review by andreea-dumitrascu — DISMISSED** [review]

File: src/Business/Detection/Common/Models/PDetection.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+        public int Score { get; set; }
+
+        public string Class { get; set; }
+
+        public DateTime Timestamp { get; set; }
+
+        public PPagination Pagination { get; set; } = new PPagination();
+    }
+}
```
Comment (RaduBuciuceanu) [inline src/Business/Detection/Common/Models/PDetection.cs:18]: As far as we are gonna use this mode for fetching only, won't be better to move it under the `Detection.Fetching.Models` namespace?

**Review by RaduBuciuceanu — CHANGES_REQUESTED** [review]

File: src/Business/Detection/Common/Models/PDetection.cs (hunk 2 above)
Comment (ConstantinescuAndrei) [reply src/Business/Detection/Common/Models/PDetection.cs]: Done

**Review by RaduBuciuceanu — DISMISSED** [review]

**Review by RaduBuciuceanu — APPROVED** [review]

**Review by danidoff — APPROVED** [review]

**Pull request merged**